components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py

```diff
@@ -61,6 +70,17 @@ def create_batch_prediction_job(
             create_batch_prediction_job_with_client, payload)
 
     # Poll batch prediction job status until "JobState.JOB_STATE_SUCCEEDED"
-    remote_runner.poll_job(get_batch_prediction_job_with_client, job_name)
+    get_job_response= remote_runner.poll_job(get_batch_prediction_job_with_client, job_name)
+    # todo remove
+    print(get_job_response)
```

Comment:
nit: testing artifact should be removed.

Reply:
done